graylog2-web-interface/src/util/PaginationHelper.js

```diff
@@ -0,0 +1,12 @@
+// @flow strict
+const PaginationHelper = {
```

Comment:
We should get rid of the enclosing object. It does not add benefit, but prevents tree shaking potentially, because we cannot import only a single function.